Content.Client/Sandbox/SandboxManager.cs

```diff
@@ -244,15 +243,15 @@ private void OnSuicideButtonClicked(BaseButton.ButtonEventArgs args)
         private void ToggleEntitySpawnWindow()
         {
             if (_spawnWindow == null)
-                _spawnWindow = new EntitySpawnWindow(_placementManager, _prototypeManager, _resourceCache, _localization);
+                _spawnWindow = new EntitySpawnWindow(_placementManager, _prototypeManager, _resourceCache);
 
             if (_spawnWindow.IsOpen)
             {
                 _spawnWindow.Close();
             }
             else
             {
-                _spawnWindow = new EntitySpawnWindow(_placementManager, _prototypeManager, _resourceCache, _localization);
+                _spawnWindow = new EntitySpawnWindow(_placementManager, _prototypeManager, _resourceCache);
```

Comment:
Remove the _loc Parameter from the Constructor

Reply:
Ok. I will submit a PR for the engine to remove _loc parameter in constructor.